source/Calamari.Shared/Calamari.Shared.csproj

```diff
@@ -61,7 +61,6 @@
     <PackageReference Include="SharpCompress" Version="0.24.0" />
     <PackageReference Include="Sprache" Version="2.3.0" />
     <PackageReference Include="System.ValueTuple" Version="4.4.0" />
-    <PackageReference Include="YamlDotNet" Version="8.1.2" />
```

Comment:
Removed this reference here because it is already included ass a transitive dependency. Removing it here avoids ambiguity.